src/test/java/eu/europeana/metis/sandbox/controller/DatasetControllerIT.java

```diff
@@ -30,12 +31,13 @@
   public void processDataset_expectStatus_accepted() throws Exception {
 
     MockMultipartFile dataset = new MockMultipartFile("dataset", "dataset.txt", "text/plain",
-        testUtils.readFileToBytes("zip/dataset-valid.zip"));
+        testUtils.readFileToBytes("zip"+File.separator+"dataset-valid.zip"));
```

Comment:
Read my comment on the metis-framework PR, if what I'm suggesting works, we could then apply it here as well and all other locations below.
https://github.com/europeana/metis-framework/pull/479